PHP Array Completion on Empty String

### Short Summary
`php-complete-array-members` triggered on empty string.

### Steps to Reproduce
Use the following PHP and type the last quote.
```php
<?php

$arr = []
$arr[] = ['
```

### Expected results
No `pystderr.log` message.

### Actual results
The following `pystderr.log` message:
```
[ERROR] koCodeIntel:     INFO: eval '' at testComplete.php#4  <Trigger 'php-complete-array-members' at 28>
    INFO: _hit_from_citdl:: expr: '', scoperef: (<blob u'testComplete.php'>, [])
    INFO: _hit_from_citdl:: expr tokens: ['']
    INFO: _hits_from_first_part:: find ' ...' starting at (<blob u'testComplete.php'>, []):
    INFO: _hits_from_first_part:: scoperef now (<blob u'testComplete.php'>, []):
    INFO: _hits_from_first_part:: checking namespace aliases
    INFO: _hits_from_first_part:: pt3: is '' accessible on (<blob u'testComplete.php'>, [])? no
    DEBUG: _hit_from_elem_imports:: Checking imports, tokens[0]: '' ... imp_elem: <blob u'testComplete.php'>
    INFO: _hit_from_elem_imports:: trying import everything: tokens: ['']
    INFO: _hit_from_elem_imports:: trying with lpath: ('',)
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP curdirlib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP curdirlib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP extradirslib> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <PHP cataloglib: jQuery> => []
    INFO: _hit_from_elem_imports:: ie: lookup ('',) in <php-7.0 stdlib> => []
    INFO: _hit_from_elem_imports:: ie: no matches found
    ERROR: evaluating '' at testComplete.php#4: could not resolve first part of ''
```

### Platform Information
Komodo Edit 11.1.2 Nightly
maxOS Mojave

--- a/src/udl/skel/PHP/pylib/lang_php.py
+++ b/src/udl/skel/PHP/pylib/lang_php.py
@@ -631,9 +631,13 @@ class PHPLangIntel(CitadelLangIntel, ParenStyleCalltipIntelMixin,
                 ctlr.error(str(ex))
                 ctlr.done("error")
                 return
-            line = buf.accessor.line_from_pos(pos)
-            evalr = PHPTreeEvaluator(ctlr, buf, trg, citdl_expr, line)
-            buf.mgr.request_eval(evalr)
+            # Don't try to find array members on empty string
+            if citdl_expr == "" and trg.form == TRG_FORM_CPLN and trg.type == "array-members":
+                ctlr.done("success")
+            else:
+                line = buf.accessor.line_from_pos(pos)
+                evalr = PHPTreeEvaluator(ctlr, buf, trg, citdl_expr, line)
+                buf.mgr.request_eval(evalr)
 
     def _citdl_expr_from_pos(self, trg, buf, pos, implicit=True,
                              include_forwards=False, DEBUG=False):

